tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -217,8 +217,6 @@ def test_blows_up_on_invalid_args(self):
     with self.assertRaises(TypeCheckError):
       self._default_args_execute_process_request(argv=('1',), env=['foo', 'bar'])
 
-    # TODO(cosmicexplorer): we should probably check that the digest info in
-    # ExecuteProcessRequest is valid, beyond just checking if it's a string.
     with self.assertRaisesRegexp(TypeCheckError, "env"):
```

Comment:
TODO no longer relevant?

Reply:
Yes, the `ExecuteProcessRequest` constructor does some type checking/coercion now.